index.html

```diff
@@ -0,0 +1,196 @@
+                <span class="navbar-toggler-icon"></span>
+              </button>
+          <div class="collapse navbar-collapse" id="navbarResponsive">
+            <ul class="navbar-nav ml-auto">
+              <li class="nav-item">
+                <a class="nav-link" href="#features">Features</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#works">How it works</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="#pricing">Pricing</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </nav>
+    </header>
+    <div class="row row-padding">
+      <div class="col-12">
+        <h4 class="alig">
+          <b>Handcrafted, home-made masterpieces</b>
+        </h4>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-4"></div>
+      <div class="col-4">
+        <form id="form" method="POST" action="https://www.freecodecamp.com/email-submit">
+          <div class="form-group">
+            <input type="email" class="form-control" name="email" id="email" placeholder="Enter email" required>
+          </div> <p></p><p></p>
+          <div class="form-group">
+            <div class="align-center">
+              <input type="submit" id="submit" class="btn btn-warning text-center"  value="GET STARTED"/>
+            </div>
+          </div>
+        </form>
+      </div>
+      <div class="col-4"></div>
+    </div> 
+    <div class="row top-padding">
+      <div class="row">
+          <div class="col-4 right-padding">
+            <i class="fas fa-fire ffire"></i>
+
+          </div>
+          <div class="col-7">
+              <div>
+                <h4><b> Premium Materials</b></h4>
+                Our trombones use the shiniest brass which is sourced locally. This will increase the longevity of your purchase.
+              </div>
+          </div>
+      </div> 
+      <a href="#features">.</a>
+      <div class="row padding-top-70" id="features">
+        <div class="col-4 right-padding-two">
+          <i class="fas fa-truck ffire"></i>
+        </div>
+        <div class="col-7">
+            <div>
+              <h4><b> 
+                Fast Shipping</b></h4>
+                We make sure you recieve your trombone as soon as we have finished making it. We also provide free returns if you are not satisfied. 
+            </div>
+        </div>
+    </div>
+      <div class="row padding-top-70">
+        <div class="col-4 padding-right">
+          <i class="fas fa-battery-full ffire"></i>
+        </div>
+        <div class="col-7">
+            <div>
+              <h4><b> Quality Assurance</b></h4>
+              For every purchase you make, we will ensure there are no damages or faults and we will check and test the pitch of your instrument. 
```

Comment:
Same as stated above, kindly do the same for all other places you must have structured this way.

Reply:
This is a good follow-up review comment. However, I suggest that you make it more descriptive 💯 for the person you are reviewing to understand what "above" you are referring to 😃

You could quote the comment you are referring to.